tests/NuGet.IndexingTests/Extraction/CatalogPackageMetadataExtractorTests.cs

```diff
@@ -23,6 +23,24 @@ public void AddsListed(object catalogEntry, string expected)
             Assert.Equal(expected, metadata["listed"]);
         }
 
+        [Theory, MemberData(nameof(AddsSemVerLevelKeyData))]
+        public void AddsSemVerLevelKey(object catalogEntry, bool expectedToContainKey, string expected)
+        {
+            // Arrange
+            var catalogEntryJObject = CatalogEntry(catalogEntry);
+
+            // Act
+            var metadata = CatalogPackageMetadataExtraction.MakePackageMetadata(catalogEntryJObject);
+
+
+            // Assert
+            Assert.Equal(expectedToContainKey, metadata.Keys.Contains("semVerLevelKey"));
```

Comment:
Use the `LuceneMetadata.SemVerLevelKeyPropertyName` constant here (and throughout these tests).